addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -3,60 +3,77 @@
+                        <tr t-if="registration.company_name"><td>Company</td><td><t t-out="registration.company_name"/></td></tr>
                         <tr><td>Event</td><td><t t-out="registration.event_display_name"/></td></tr>
-                        <tr t-if="!registration.event_id and registration.company_name"><td>Company</td><td><t t-out="registration.company_name"/></td></tr>
-                        <tr><td>Name</td><td><t t-out="registration.name"/></td></tr>
-                        <tr t-if="registration.ticket_name"><td>Ticket</td><td><t t-out="registration.ticket_name"/></td></tr>
-                        <tr t-if="registration.sale_status_value"><td>Payment</td><td><t t-out="registration.sale_status_value"/></td></tr>
-                        <tr t-if="registration.registration_answers">
-                            <td>Answers</td>
-                            <td>
-                                <span t-foreach="registration.registration_answers" t-as="registration_answer" t-key="registration_answer_index"
-                                    t-attf-class="o_tag o_tag_badge_text o_tag_color_#{registration_answer_index % 10} badge rounded-pill p-1 me-1 mb-1"
-                                    t-out="registration_answer"/>
-                            </td>
-                        </tr>
+                        <tr t-if="registration.ticket_name"><td>Ticket Type</td><td><t t-out="registration.ticket_name"/></td></tr>
+                        <t t-if="registration.registration_answers">
+                            <tr t-if="registration.registration_answers.length > 0">
+                                <td class="d-flex">Answers</td>
+                                <td>
+                                    <div class="d-flex flex-wrap p-0">
+                                        <span t-foreach="registration.registration_answers" t-as="registration_answers" t-key="registration_answers_index"
+                                        t-attf-class="o_tag o_tag_badge_text o_tag_color_#{registration_answers_index % 10} badge rounded-pill text-truncate p-1 me-1 mb-1"
+                                        t-out="registration_answers"/>
+                                    </div>
+                                </td>
+                            </tr>
+                        </t>
                     </table>
                 </div>
             </div>
-
+            <div class="mt-4">
+                <span class="fs-2 link-success" role="button" t-on-click="() => this.onRegistrationPrintPdf()">
+                    <i class="fa fa-solid fa-print"/><span > Print Ticket</span>
+                </span>
+            </div>
+            <div>
+                <span class="fs-2 link-success" role="button" t-on-click="() => this.onRegistrationView()">
+                    <i class="fa fa-regular fa-file"/><span> Open Details</span>
+                </span>
+            </div>
```

Comment:
just asking, should we put them in the same div ? Not saying we have to, but if we can simplify, it could make sense